packs/google/requirements.txt

```diff
@@ -0,0 +1 @@
+-e git+https://github.com/Kami/Google-Search-API.git@stackstorm_patched#egg=Google-Search-API
```

Comment:
Sadly both of the original packages were broken in one way or another.

I've pushed fixes to that branch. I also submitted an upstream PR, but I have no idea when and if it will get merged...